.github/workflows/ci.yaml

```diff
@@ -292,81 +292,37 @@ jobs:
           go-version: "1.20.0-rc.3"
       - run: |
           go version
-      - run: |
-          cd mirrord/layer/tests/apps/self_open
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/fileops/go
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/issue834
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/dir_go
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/dir_go_bypass
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/read_go
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/write_go
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/lseek_go
-          go build -o 20
-      - run: |
-          cd mirrord/layer/tests/apps/faccessat_go
-          go build -o 20
+      - run: | # Build Go test apps.
+          for go_mod in $(find . -name "go\.mod")
```

Comment:
this is great. maybe we should add it to `scripts/build_tests.sh` or something similar os we can do it easily locally too?

Reply:
Done ^^